Sources/SwiftDocC/Infrastructure/Topic Graph/TopicGraph.swift

```diff
@@ -98,7 +98,10 @@ struct TopicGraph {
         
         /// If true, the topic has been manually organized into a topic section on some other page.
         var isManuallyCurated: Bool = false
-        
+
+        /// For overloaded symbols, the reference to their overload group node.
+        var overloadGroup: ResolvedTopicReference? = nil
```

Comment:
I could imagine having two booleans, one for being an overload group and one for being an overload; that way the overload group could crawl its children for those with the latter flag, and they wouldn't need to store the reference, and then we could still preserve any other topic groups that exist on the overload group.

Reply:
Oh. I hadn't thought about there being curation in the doc document that the overload group gets from one of the overloaded symbols. 

I don't _think_ we need a second boolean. It should be sufficient to filter out `isManuallyCurated`. That would probably also help avoid including one of the overloaded symbols if it's already manually curated.

Reply:
It could also be good to have a test that verifies that the overload group lists task groups for both the manually curated symbols and the overloaded symbols.

Reply:
I hadn't thought about using `isManuallyCurated`, thanks for that!

I've modified the check based on that, and added a test for overload groups with manual curation.